### ISSUE TITLE
Double Scanning

### ISSUE THREAD

**Issue by jcuenod** [wc-library/selfcheck#12]
Scanning twice produces two rows in the output table which freaks out patrons (and it's not clear whether it's now checked back in or something).

**Comment by jeffnm** [comment]
Are these double scans that happen in rapid succession, or is the item being scanned twice with a pause between? 

I would expect an item that's already been checked out to return an error that the item can't be checked out, and directing the user to ask the circulation desk for help.

**Comment by jeffnm** [comment]
Double scanning turns out to be the API "Renewing" the loan. 

For example:

```
{ 
  loan_id: 12345,
  ...,
  last_renew_date: '2020-09-30T18:37:10.982Z',
  last_renew_status: {
    value: '401170;',
    desc: 'The due date is already set to the renewed due date'
  }
}
```
This still is a valid Loan Item, so the current client handling is to add the relevant data to the table.  

### Proposed solution

Check the values of a successful Loan Item response against the values already received during the session. If it's the same, ignore it.

**Comment by jeffnm** [comment]
I can confirm this double scanning is still an issue in #14 
You don't have to fix it in #14, but I'd still like it fixed per the proposal https://github.com/wc-library/selfcheck/issues/12#issuecomment-701574590

**Comment by jcuenod** [comment]
https://github.com/wc-library/selfcheck/pull/16

**Comment by jcuenod** [comment]
I believe this issue can be closed

**Comment by jeffnm** [comment]
Addressed in #19

**Comment by jcuenod** [comment]
(and #16)
